From a167c9fe49936e9ac4b8982e6fdfa6cc40577602 Mon Sep 17 00:00:00 2001
From: Kemal Rasim Sh <kshakir@codeaurora.org>
Date: Tue, 17 Nov 2020 11:32:32 +0200
Subject: [PATCH 1/1] Disable logging on perf builds

Disable logging on perf builds in order not to
run out of storage space over the time.

Signed-off-by: Kemal Rasim Sh <kshakir@codeaurora.org>
---
 gstd/gstd_log.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gstd/gstd_log.c b/gstd/gstd_log.c
index c678bd0..e187e27 100644
--- a/gstd/gstd_log.c
+++ b/gstd/gstd_log.c
@@ -32,7 +32,7 @@ GST_DEBUG_CATEGORY (gstd_debug);
 #define GSTD_LOG_NAME "gstd.log"
 #define GST_LOG_NAME "gst.log"
 #define GSTD_DEBUG_PREFIX "gstd"
-#define GSTD_DEBUG_LEVEL "WARNING"
+#define GSTD_DEBUG_LEVEL "none"
 
 static void
 gstd_log_proxy (GstDebugCategory * category, GstDebugLevel level,
--
2.17.1
